MetaMask/metamask-extension
# feat: Icon Button to use light theme in dark mode

Here’s the reworded PR template for the `IconButton` component:

---

## **Description**

This PR updates the `IconButton` component (also known as `ActionButton`) to use light theme colors when rendered in dark mode. This update is part of the ongoing brand evolution, ensuring that the visual design is consistent with the new brand guidelines. The primary objective is to align the dark mode appearance of this component with the light theme color palette, improving overall user experience and maintaining visual coherence across the application.

## **Related issues**

Fixes: 735

## **Manual testing steps**

1. Navigate to any page where the `IconButton` component is rendered.
2. Enable dark mode in the application.
3. Observe the color scheme of the `IconButton` component to ensure it uses the updated light theme colors.
4. Compare the colors with the brand guidelines provided.
5. Test across various instances of the `IconButton` component in dark mode to ensure consistency.

## **Screenshots/Recordings**

### **Before**
<img width="909" alt="Screenshot 2024-09-03 at 5 04 44 PM" src="https://github.com/user-attachments/assets/338cf8a5-0ae7-4cc1-99ca-0d1b2627da2f">
<img width="858" alt="Screenshot 2024-09-03 at 5 03 40 PM" src="https://github.com/user-attachments/assets/e10d9119-536b-485d-96d0-7aab1f14efe1">


### **After**
<img width="973" alt="Screenshot 2024-09-03 at 5 05 26 PM" src="https://github.com/user-attachments/assets/e62b1bf5-c109-4d42-87e0-47a358f7184e">
<img width="732" alt="Screenshot 2024-09-03 at 5 03 48 PM" src="https://github.com/user-attachments/assets/c6ccd9a2-068b-4e16-aae0-00ab34108537">



## **Pre-merge author checklist**

- [x] I’ve included tests where applicable.

## **Pre-merge reviewer checklist**


---

This version is tailored specifically for the `IconButton` component, reflecting the necessary changes and testing steps.
